values/src/main/java/org/spine3/net/Urls.java

```diff
@@ -20,12 +20,24 @@
 
 package org.spine3.net;
 
+/**
+ * Utility class, which simplifies working with {@link Url}.
+ * <p>
```

Comment:
Please remove the paragraph tag. We use empty lines after the first sentence in Javadoc.